packages/near-lake-primitives/src/types/block.ts

```diff
@@ -120,6 +180,11 @@ export class Block {
 }
 
 
+/**
+ * Replacement for `BlockHeaderView` from `near-primitives`. Shrunken and simplified. 
+ *
+ * **Note:** the original `BlockHeaderView` is still accessible via the `.streamerMessage` attribute.
+ */
```

Comment:
This is how this section shows up in the docs.

I think all the details are not being shown properly?

![image](https://github.com/near/near-lake-framework-js/assets/25015977/34440d08-7ebd-4333-8c08-282b8c76cdc4)

Reply:
I think what is actually happening is that `BlockHeaderView` and the other structures below that are being pulled from `src/types/core/types.ts` and these files do not have any annotations.

The point of the library was to abstract away these core structures so I am not sure what value it brings to show the user of the library this. If we decide to include this in the docs, we should make sure the developer knows that these are `low level` structures that they do not neccesarily need to know about, but currently they do not get any context about what is in these structures in the first place.

![image](https://github.com/near/near-lake-framework-js/assets/25015977/3e60628f-c4e0-4e83-9dff-e0555bab44ee)

Reply:
Alright, thanks for sharing the context of this library. 
Let me check the TypeDocs settings and see if I can remove these low-level structures from the output html.